src/v/security/audit/schemas/tests/ocsf_schemas_test.cc

```diff
@@ -0,0 +1,577 @@
+#include "json/reader.h"
+#include "json/stream.h"
+#include "json/stringbuffer.h"
+#include "security/audit/schemas/application_activity.h"
+#include "security/audit/schemas/iam.h"
+#include "security/audit/schemas/schemas.h"
+
+#include <seastar/testing/thread_test_case.hh>
+
+namespace sa = security::audit;
+
+inline ss::sstring minify(std::string_view json) {
+    ::json::Reader r;
+    ::json::StringStream in(json.data());
+    ::json::StringBuffer out;
+    ::json::Writer<::json::StringBuffer> w{out};
+    r.Parse(in, w);
+    return ss::sstring{out.GetString(), out.GetSize()};
+}
```

Comment:
This looks familiar ;')

Reply:
Probably a candidate to move to some json utility lib